src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -1448,6 +1449,45 @@ void ov::npuw::util::gather(const ov::SoPtr<ov::ITensor>& src,
     }
 }
 
+ov::SoPtr<ov::ITensor> ov::npuw::util::view(const ov::SoPtr<ov::ITensor>& src,
+                                            const ov::npuw::util::View& from,
+                                            const ov::npuw::util::View& to) {
+    const auto type = src->get_element_type();
+    NPUW_ASSERT(from.size() == to.size());
+
+    // Sub-byte views are not supported here
+    NPUW_ASSERT(type != ov::element::u4 && type != ov::element::i4);
```

Comment:
Wouldn't it be needed when we do spatial for subgraphs with weights? Do you mind adding a FIXME here?

Reply:
there's no difference. We only tile activations, we don't tile weights. So subgraphs weights will work over activation ranges.